packages/events/src/modules/bicho/finishBets.ts

```diff
@@ -64,33 +76,15 @@ const didUserWin = (results: number[][], option: string, bet: BichoBetType): boo
       return results.some((a) => `${a[1]}${a[2]}${a[3]}` === option);
     case 'thousand':
       return results.some((a) => `${a[0]}${a[1]}${a[2]}${a[3]}` === option);
-    case 'animal': {
-      const animals = results.map((a) => {
-        const ten = Number(`${a[2]}${a[3]}`);
-        return BICHO_ANIMALS[Math.floor(ten / 4)];
-      });
+    case 'animal':
       return animals.some((a) => a === option);
-    }
-
-    case 'corner': {
-      const animals = results.map((a) => {
-        const ten = Number(`${a[2]}${a[3]}`);
-        return BICHO_ANIMALS[Math.floor(ten / 4)];
-      });
-      const userChoices = option.split(' | ');
-      return animals.every((a) => userChoices.includes(a));
-    }
-    case 'sequence': {
-      const animals = results.map((a) => BICHO_ANIMALS[Math.floor(Number(`${a[2]}${a[3]}`) / 4)]);
-      const user = option.split(' | ');
-      const hasTwoAnimals = user.every((a) => animals.includes(a));
-      if (!hasTwoAnimals) return false;
-
-      const firstIndex = animals.indexOf(user[0]);
-      const secondIndex = animals.indexOf(user[1]);
-
-      return firstIndex < secondIndex;
-    }
+    case 'corner':
+      return hasTwoAnimals(animals, option.split(' | '));
```

Comment:
The test suits failed for this.

Looking at the old code, the corner case uses the loop
```ts
animals.every((a) => userChoices.includes(a));
```
While the sequence loop (hasTwoAnimals) use 
```ts
user.every((a) => animals.includes(a));
```

The function `hasTwoAnimals` is the old sequence loop. The corner loop needs to be different.


The test case which return an error was the check `vaca | vaca | vaca | vaca | vaca` in an array of results `vaca | águia | veado | leão | galo`

Reply:
Got it. In fact, the code has this inconsistency. I believe that because `hasTwoAnimals()` follows a pattern, this modification will have to be directly in `switch`. I will do it here.